### ISSUE TITLE
Feature request: possibility to manually modify mac address shown in details of the device

### ISSUE THREAD

**Issue by tradexsrl** [Ylianst/MeshCentral#1102]
Hi , will be possible to add the possibility to manually modify the mac address shown in the details device ?
my case is: on our pc we have as main OS openxt and then a windows 7 vm on top of that.
with agent installed into the vm , you retrieve the virtual mac address generated by openxt vm configuration and not the real hardware mac address of the net , so is not possible to use the wakeup action.
if i can manually modify what is shown , i can supply the real one and do wake up action:
![immagine](https://user-images.githubusercontent.com/62424179/78219246-72b10200-74bf-11ea-839e-569d28a06d31.png)

**Comment by Ylianst** [comment]
Oh, this is interesting. I never expected this case. What I could do is allow you to add an extra "Wake MAC" for a device. I still do what to display the network interface information accurately.

**Comment by tradexsrl** [comment]
so i can select on which mac address send the wake up action...perfect :-)

**Comment by Ylianst** [comment]
Yes, or rather, specify an added one. It would work for this. However, I have lots going or, not sure when I will get around to this.

**Comment by si458** [comment]
This feature hasn't been implemented but
you can use the wol feature from the console tab now!
Pick a device that's in the same group as ur offline device, and also on the same network for the wol broadcast packets
Go into its console tab
Then type `wakeonlan MACADDRESSWITHOUTCOLONS`

**Comment by tradexsrl** [comment]
I'm not sure to have understood... if I open a console on a device which is online and send the command as you described, it won't work because the MAC address shown is the MAC address of the virtual machine created in openxt hypervisor and not the hardware MAC address of the computer ....

**Comment by si458** [comment]
so you would need to manually type in the mac address of the remote computer thats on the host and not the VM,
its a work around but it should work for you, if you can already wake on lan using the HOSTS mac address instead of the VM mac address

**Comment by tradexsrl** [comment]
Also to note that many times all pc are powered off when holidays and I do not install agent on main servers

**Comment by tradexsrl** [comment]
But I need to have an excel file with all mac addresses related to the computers....instead of finding inside the meshcentral portal

**Comment by si458** [comment]
what about the notes section? could you save them in there?
you can also now set `showNotesPanel: true` inside domains in config.json, this will show the notes panel next to the ICON on the device https://github.com/Ylianst/MeshCentral/issues/2513#issuecomment-1807181669

**Comment by tradexsrl** [comment]
As a workaround I rdp into a server and with wakeonlan external tool I send magic packet to the host MAC address taken from an excel file...as you can see is not a straightforward task , instead of those proposed by @Ylianst

**Comment by si458** [comment]
as suggested above, save mac into notes of that device, and check notes as a work around https://github.com/Ylianst/MeshCentral/issues/1102#issuecomment-1878635496
but sadly development has grinded to a halt at the moment, im in process of working my way through things i can fix/implement like GUI things, but when it comes to database stuff, i tend to stay away at the moment, as this feature would require saving extra information into the database,
but this might change in the future tho, you never know,
i will keep it open tho, and look into it in the coming months

**Comment by tradexsrl** [comment]
Many thanks
